mestopy/mestopy.py

```diff
@@ -1 +1,242 @@
-"""Main module."""
+import numpy as np
+from pyfar import Signal
+
+
+# Class to generate ref-Objects, that can bei part of the MeasurementChain
+class DeviceObj(object):
+    """Class for device in MeasurementChain.
+
+    This class holds methods and properties of a device in the
+    'MeasurementChain' class.
+    """
+
+    def __init__(self, data, sens=1, name=''):
+        """Init DeviceObj with data.
+
+        Attributes
+        ----------
+        data : Signal
+            Signal with the inverted frequency response of the device.
+        sens : float
+            Sensitivity of the device as a factor.
+        name : str
+            Name of the device.
+        """
+        self.sens = sens
+        self.name = name
+        self.data = data
```

Comment:
You could add checks, e.g., `if  instance(name, str)` and raise ValueErrors if the checks fail